src/Owin.Security.Saml/SamlMessage.cs

```diff
@@ -229,4 +229,4 @@ public override string BuildFormPost()
             return base.BuildFormPost(); // See Saml20SignonHandler.cs, line 591 (post binding)
         }
     }
-}
\ No newline at end of file
+}
```

Comment:
Not sure where this came from. I even tried to reverse it but I think that the web interface might add it in by default.